packages/esbuild-plugin/src/index.ts

```diff
@@ -50,6 +51,12 @@ function esbuildDebugIdInjectionPlugin(): UnpluginOptions {
 
     esbuild: {
       setup({ initialOptions, onLoad, onResolve }) {
+        if (initialOptions.bundle) {
+          logger.warn(
+            "Esbuild's `bundle` option is currently not supported! Esbuild will probably crash now."
```

Comment:
l: Did I understand the issue correctly that this only happens for `bundle: true`? In this case, maybe we tell users this specifically:

```suggestion
            "Esbuild's `bundle: true` option is currently not supported! Esbuild will probably crash now."
```

Also, is there anything we can recommend them here instead of just saying that their build will crash?

Reply:
Absolutely valid point. I updated the wording!